lib/config/rules/jest.js

```diff
@@ -0,0 +1,20 @@
+// see https://github.com/sindresorhus/eslint-plugin-ava
+
+module.exports = {
+  // Disallow all forms of skipped tests (`.skip`, prepending `x` to test helpers, empty function bodies, `pending()` calls.)
+  'jest/no-disabled-tests': 'error',
```

Comment:
Pretty sure this will generate enough complaints that we'll end up disabling it :|

Reply:
I think this is good. Skipped tests are _never fixed_ so they should just be removed.

Reply:
We have a bunch of skipped tests and I agree that they are just never actually filled out, so I am fine with this.